strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/validation/groupId/MavenGroupIdLowercaseValidator.java

```diff
@@ -0,0 +1,46 @@
+package org.carlspring.strongbox.storage.validation.groupId;
+
+import org.carlspring.strongbox.artifact.coordinates.MavenArtifactCoordinates;
+import org.carlspring.strongbox.providers.ProviderImplementationException;
+import org.carlspring.strongbox.providers.io.RepositoryFileAttributes;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.carlspring.strongbox.storage.validation.exceptions.LowercaseValidationException;
+
+import java.io.IOException;
+import java.nio.file.Files;
+
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by dinesh on 12/6/17.
+ */
+@Component("GroupIdCaseValidator")
```

Comment:
I'd say this should be `mavenGroupIdLowercaseValidator`. Also, the convention we follow for Spring is camel hump.

Reply:
Agree. I actually saw it one of the packages so thought that this is the conventions used in this project :-)